audiomentations/augmentations/spectrogram_transforms.py

```diff
@@ -80,6 +81,10 @@ class SpecChannelShuffle(BaseSpectrogramTransform):
     supports_multichannel = True
     supports_mono = False
 
+    def __init__(self):
+        super().__init__()
+        self.name = "SpecChannelShuffle"
+
```

Comment:
I think you have to revert this change, because I see some test failures like this now:

```
        augmenter = SpecCompose(
            [
>               SpecChannelShuffle(p=1.0),
                SpecFrequencyMask(p=1.0),
            ]
        )
E       TypeError: __init__() got an unexpected keyword argument 'p'
```

Reply:
Yes, I wanted to revert it, I forgot to do it. I'm fixing it